metabase/metabase
# Add scoped search in the Search tab in Entity Picker

Closes #47291

### How to verify



- first folder (i.e. a collection or a database) in each tab (except "Recents" and "Search") should automatically be selected when given tab is opened for the first time (unless entity picker already has a value)
- tab state should be preserved when navigating between tabs
- search is scoped to a given folder (non-recursive) when a folder is selected and a given tab is opened

This change affects all entity pickers (data picker, question picker, collection picker, dashboard picker).

### Demo

https://github.com/user-attachments/assets/de2d9cf3-217e-46c7-b90e-df4bd04a31a2
